plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -2685,8 +2685,6 @@ function wc_update_890_update_connect_to_woocommerce_note() {
 		return;
 	}
 	Notes::delete_notes_with_name( WooSubscriptionsNotes::CONNECTION_NOTE_NAME );
-	$new_note = WooSubscriptionsNotes::get_note();
-	$new_note->save();
```

Comment:
Need to update the doc comment too (i.e. if it exists, delete the note only).

Reply:
AFAIK, this func would run when a site is updated to 8.9.0. Therefore, anyone updated to 9.0 from 8.9.x wouldn't run `wc_update_890_update_connect_to_woocommerce_note`.

Reply:
If we'd like to remove the note for anyone updated to 9.0 from 8.9+, it should be something like https://github.com/woocommerce/woocommerce/commit/363d761eae1efcfe1436bd5500984f1b3fd0a1ef.